src/command.rs

```diff
@@ -14,17 +14,22 @@
 
 use crate::{
     credentials::Credentials,
+    generator,
+    generator::Language,
```

Comment:
This needs to be feature guarded to work when the build feature is disabled.

Try running `cargo build --no-default features` to see what needs to be changed

Reply:
Thanks for this tip, using that I was able to resolve some unused imports too